draft-birkholz-scitt-architecture.md

```diff
@@ -235,31 +235,44 @@ Reputable Issuers are thus incentivized to carefully review their Statements bef
 
 # Architecture Overview
 
-~~~~
-             Artifact
-                |
-                v                      +------------------+
- Issuer    -> Statement    Envelope    | DID Key Manifest |
-                \           /          | (decentralized)  |
-                 \         /           +------------------+
-                  \ ______/               |     |
-                      |                   |     |
-                      v        signature  |     |
-                    Claim  <--------------/     |
-                      |                         |
-                      |  Claim      +---------+ |
-                      |------------>| Trans.  | |
-Transparency ->       +<------------| Registry| /
-Service               |  Receipt    +--------+ X
-                      v                       / \
-                 Transparent                 /   \
-                    Claim                   /    |
-                      |\                   /     |
-                      | \                 /      |
-                      |  \               /       |
-Verifier    ->        |    Verify Claim          |
-                      |                          |
-Auditor    ->       Collect Receipts     Replay Registry
+~~~~ aasvg
+                    .----------.
+                   |  Artifact  |
+                    '----+-----'
+                         v
+                    .----+----.  .----------.
+Issuer       -->   | Statement ||  Envelope  |
+                    '----+----'  '-----+----'
+                         |             |       +------------------+
+                          '----. .----'        | DID Key Manifest |
+                                |              | (decentralized)  |
+                                v              +---+----------+---+
+                             .--+--.   Sign Claim  |          |
+                            | Claim +<------------'           |
+                             '--+--'                          |
+                                |            +--------------+ |
+                             .-' '---------->+ Transparency | |
+                            |   .-------.    |              | |
+Transparency -->            |  | Receipt |<--+   Registry   | |
```

Comment:
We discussed a future revision where claim and receipt are visualized next to each other.

Reply:
https://github.com/ietf-scitt/draft-birkholz-scitt-architecture/issues/32